PhysicsTools/PyTorch/interface/TorchCompat.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PhysicsTools_PyTorch_interface_TorchCompat_h
```

Comment:
Why is this called "Compact"? I think just Torch.ch would be better.

Reply:
Just a suggestion, how about renaming this to `TorchInterface.h` ?